addons/binding/org.openhab.binding.keba/ESH-INF/thing/kecontact.xml

```diff
@@ -135,5 +136,11 @@
 		<description>Power consumption in Wh. Total energy consumption is added up after each completed charging session</description>
 		<state pattern="%d Wh" readOnly="true"></state>
 	</channel-type>
+    <channel-type id="display" advanced="true">
```

Comment:
I saw 1, which I fixed manually. I guess you are telling me there is second one ;-)